app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -192,6 +171,40 @@ fun ProgressScreen(
   }
 }
 
+@Composable
+fun AchievementColumn(achievements: List<String>) {
+
+  if (achievements.isEmpty()) {
+
+    Text(
+        text = "You don't have any achievements yet!",
```

Comment:
Suggestion: As per the coach feedback, all hardcoded strings will have to be moved to the `strings.xml`, might be nice to move it now to make refactoring easier later

Reply:
You are right, it is done thank you!